html-forms/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>html-forms</title>
+  </head>
+  <body>
+    <h1> Coding Form!</h1>
+    <form>
+      <div class="section">
+        Email
+        <label>
+          <input type="text" name="email"/>
+        </label>
+      </div>
+      <div class="section">
+        Why is Coding cool?<br>
+        <select name="dropdown">
+          <option value="select an option">Select an option</option>
+          <option value="because it's interesting">Because it's interesting</option>
+          <option value="because it's fun">Because it's fun</option>
+          <option value="because it's powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <div class="section">
+        <div class="text">
+          Favorite Language (choose one)
+        </div>
+        <label>
+          <input type="radio" name="language" value="HTML"/> HTML
+        </label>
+        <label>
+          <input type="radio" name="language" value="CSS"/> CSS
+        </label>
+        <label>
+          <input type="radio" name="language" value="Javascript"/> JavaScript
+        </label>
+      </div>
+      <div class="section">
+        <div class="text">
+          Would you like to sign up for more information?
+        </div>
+        <label>
+          <input type="checkbox" name="sign-up" value="yes"/>
+        </label>
+        Sign me up!
+      </div>
+      <div class="section">
+        <div class="text">
+          Comments
+        </div>
+        <textarea name="comments" cols="30" rows="10"></textarea>
```

Comment:
`textarea` needs a `label`. I would move the text on line 49 inside a `label` and link the two using a `for` attribute.